apps/cookbook/src/app/shared/api-description/api-description.shared.scss

```
@@ -0,0 +1,32 @@
+@import './libs/designsystem/src/lib/scss/utils';
+
+.api-description {
```

Consider adding `table` as most of the styling depends on this class being set on a table:
```suggestion
table.api-description {
```

---

I think that it unnecessarily increases the selector's specificity, but in this case it doesn't make much of a difference, so I'm ok with adding `table`. Alternatively the class selector could be renamed to `.api-description-table`, but that would require a bit more work.

---

I agree 👍🏻